### ISSUE TITLE
Feature: generate all output formats when using JSON-resume themes

### ISSUE THREAD

**Issue by herau** [hacksalot/HackMyResume#80]
When i'm using json resume theme, the pdf version isn't generated:

```
$ hackmyresume BUILD resume.json to out/resume.all -t node_modules/jsonresume-theme-elegant-leonth

*** HackMyResume v1.3.0 ***
Applying JSONRESUME-THEME-ELEGANT-LEONTH theme (1 formats)
Reading resume: resume.json
Generating HTML resume: out/resume.html
Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
Error
    at Function.createFromInputFallback (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:746:36)
    at configFromString (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:826:32)
    at configFromInput (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:1353:13)
    at prepareConfig (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:1340:13)
    at createFromConfig (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:1307:44)
    at createLocalOrUTC (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:1385:16)
    at local__createLocal (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:1389:16)
    at utils_hooks__hooks (/Volumes/diskE/Users/n27/Documents/resume/node_modules/moment/moment.js:16:29)
    at humanizeDate (/Volumes/diskE/Users/n27/Documents/resume/node_modules/jsonresume-theme-elegant-leonth/index.js:18:10)
    at processDates (/Volumes/diskE/Users/n27/Documents/resume/node_modules/jsonresume-theme-elegant-leonth/index.js:28:40)
    at Function._.each._.forEach (/Volumes/diskE/Users/n27/Documents/resume/node_modules/underscore/underscore.js:153:9)
    at JRSTheme.render (/Volumes/diskE/Users/n27/Documents/resume/node_modules/jsonresume-theme-elegant-leonth/index.js:32:4)
    at single (/usr/local/lib/node_modules/hackmyresume/src/verbs/generate.js:148:31)
    at /usr/local/lib/node_modules/hackmyresume/src/verbs/generate.js:78:17
    at Array.forEach (native)
    at build (/usr/local/lib/node_modules/hackmyresume/src/verbs/generate.js:77:13)
Generating JSON resume: out/resume.json
Generating YML resume: out/resume.yml
Generating PNG resume: out/resume.png
```

**Comment by hacksalot** [comment]
Added to next release, thanks.

**Comment by hacksalot** [comment]
Still in progress. Slated for v1.5.

**Comment by hacksalot** [comment]
See also #96.

**Comment by hacksalot** [comment]
Fixed in the latest version ([v1.5.0](https://github.com/hacksalot/HackMyResume/releases/tag/v1.5.0)). HMR will now attempt to generate a PDF for JSON Resume themes. Feel free to log a new issue or comment here if you still get errors. We'll iron out any generation glitches caused by running wkhtmltopdf/Phantom locally (instead of over a local HTTP connection) in 1.5.1 or 1.6.0. Thanks.

**Comment by herau** [comment]
thanks :-) i will try it next week !
